plugins/package-curation-providers/api/src/main/kotlin/SimplePackageCurationProvider.kt

```diff
@@ -21,11 +21,33 @@ package org.ossreviewtoolkit.plugins.packagecurationproviders.api
 
 import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.PackageCuration
+import org.ossreviewtoolkit.plugins.api.PluginDescriptor
 
 /**
  * A [PackageCurationProvider] that provides the specified [packageCurations].
  */
-open class SimplePackageCurationProvider(val packageCurations: List<PackageCuration>) : PackageCurationProvider {
+open class SimplePackageCurationProvider(
+    override val descriptor: PluginDescriptor,
+    val packageCurations: List<PackageCuration>
+) : PackageCurationProvider {
+    constructor(packageCurations: List<PackageCuration>) : this(
+        descriptor = SimplePackageCurationProvider.descriptor,
+        packageCurations = packageCurations
```

Comment:
Nit: Consider dropping the named parameters as the match the assigned variables, and other secondary constructors in this commit also don't use named parameters.